modules/example/js/multistep-display-ajax.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file
+ *
+ * Adds handling of click event for view in order to demonstrate Ajax actions
+ * for multistep selection display.
+ */
+
+(function ($) {
+
+  'use strict';
+
+  /**
+   * Attaches the behavior of the media entity browser view.
+   */
+  Drupal.behaviors.multiStepAjaxSelection = {
+    attach: function () {
+      jQuery('.view-id-files_entity_browser_ajax td.views-field-filename')
```

Comment:
This seems coupled to a specific view, and assumes that a table is used. Can we make this logic more generic?